presto-product-tests-launcher/src/main/java/io/prestosql/tests/product/launcher/cli/Launcher.java

```diff
@@ -155,7 +155,10 @@ protected Extensions getExtensions()
         @Override
         protected Object[][] getContents()
         {
-            return new Object[][]{{"project.version", readProjectVersion()}};
+            return new Object[][]{
```

Comment:
Missing space before `{`

Reply:
Sure. It's not checked by checkstyle :(

Reply:
Interesting, I assumed it was and this would fail.